docs/third-party-developers/extensibility/data-store/cart.md

```diff
@@ -44,6 +67,365 @@ To utilize this store you will import the `CART_STORE_KEY` in any module referen
 const { CART_STORE_KEY } = window.wc.wcBlocksData;
 ```
 
+## Actions
+
+### setCartData
+
+This action is used to set the cart data in the store.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartData: Object`: The new cart data.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( setCartData( newCartData ) );
+```
+
+### setErrorData
+
+This action is used to set the error data in the store.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `errorData: Object`: The new error data.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( setErrorData( newErrorData ) );
+```
+
+### receiveCartContents
+
+This action returns an action object used in updating the store with the provided cart. It omits the customer addresses so that only updates to cart items and totals are received.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartContents: Object`: The new cart contents.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( receiveCartContents( newCartContents ) );
+```
+
+### receiveApplyingCoupon
+
+This action returns an action object used to track when a coupon is applying.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `couponCode: string`: The code of the coupon being applied.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( receiveApplyingCoupon( couponCode ) );
+```
+
+### receiveRemovingCoupon
+
+This action returns an action object used to track when a coupon is removing.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `couponCode: string`: The code of the coupon being removed.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( receiveRemovingCoupon( couponCode ) );
+```
+
+### receiveCartItem
+
+This action is used to update a specific item in the cart.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `CartResponseItem: `:  A cart item API response.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( receiveCartItem( CartResponseItem ) );
+```
+
+### itemIsPendingQuantity
+
+This action returns an action object to indicate if the specified cart item quantity is being updated.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartItemKey: string`: The key of the cart item.
+-   `isPending: boolean`: The new status of whether the cart item quantity is being updated. Defaults to `true`.
```

Comment:
As mentioned earlier, for consistency-reasons, I'd use the same format we're using for our other docs. Here's an example of how other docs display default values and allowed values: https://github.com/woocommerce/woocommerce-blocks/blob/6960ae656112b5a5249673db2aaa80e13c4b94c7/docs/third-party-developers/extensibility/checkout-block/available-filters/coupons.md?plain=1#L149-L155